[2025-03-05T20:18:04.210+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T20:18:04.218+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_stock_data.fetch_stock_data scheduled__2025-03-04T09:00:00+00:00 [queued]>
[2025-03-05T20:18:04.221+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_stock_data.fetch_stock_data scheduled__2025-03-04T09:00:00+00:00 [queued]>
[2025-03-05T20:18:04.222+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-05T20:18:04.228+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): fetch_stock_data> on 2025-03-04 09:00:00+00:00
[2025-03-05T20:18:04.231+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=330) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-05T20:18:04.232+0000] {standard_task_runner.py:72} INFO - Started process 335 to run task
[2025-03-05T20:18:04.232+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_stock_data', 'fetch_stock_data', 'scheduled__2025-03-04T09:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/extract_stock_data.py', '--cfg-path', '/tmp/tmp2n6doqcc']
[2025-03-05T20:18:04.235+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask fetch_stock_data
[2025-03-05T20:18:04.257+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_stock_data.fetch_stock_data scheduled__2025-03-04T09:00:00+00:00 [running]> on host 83a9dc73146e
[2025-03-05T20:18:04.286+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_stock_data' AIRFLOW_CTX_TASK_ID='fetch_stock_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T09:00:00+00:00'
[2025-03-05T20:18:04.287+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-05T20:18:04.288+0000] {logging_mixin.py:190} INFO - YF.download() has changed argument auto_adjust default to True
[2025-03-05T20:18:04.289+0000] {cache.py:130} INFO - Failed to create TzCache, reason: Error creating TzCache folder: '/home/***/.cache/py-yfinance' reason: [Errno 17] File exists: '/home/***/.cache/py-yfinance'. TzCache will not be used. Tip: You can direct cache to use a different location with 'set_tz_cache_location(mylocation)'
[2025-03-05T20:18:04.701+0000] {logging_mixin.py:190} WARNING - [*********************100%***********************]  1 of 1 completed
[2025-03-05T20:18:04.843+0000] {logging_mixin.py:190} WARNING - [*********************100%***********************]  1 of 1 completed
[2025-03-05T20:18:04.846+0000] {python.py:240} INFO - Done. Returned value was: ['/tmp/AAPL_data.csv', '/tmp/NVDA_data.csv']
[2025-03-05T20:18:04.855+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-05T20:18:04.856+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_stock_data, task_id=fetch_stock_data, run_id=scheduled__2025-03-04T09:00:00+00:00, execution_date=20250304T090000, start_date=20250305T201804, end_date=20250305T201804
[2025-03-05T20:18:04.869+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T20:18:04.877+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-05T20:18:04.877+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
